run_npex.ipynb

```diff
@@ -0,0 +1,311 @@
p07RMtsSvCWI9p9kXjUbbfZe1gQJ6hbE4cgSTAhXaBq9JjfJMm1fC1YsyqIp0AlK/YbMtAGq1VYa6aTZpmrYMGJxnQpUjUGbOpaF8hyuTcG1imC2aVIXYa/onxm5bO2nbSbDXEoMmv/Fuljt7yItUtSCIhNrnaM6OOqa8Mat0aeT9dYkwiOW1zCIKpa9yvUXN/MXuHLf+jzlnTTCTTcaiOltVpdT6JknTAsymlsc1f3f1xwQNKXDB9hhhqOWc4XO53lmGSQPrN/jqi1pXIs5izjAn8MKAzdSeG8aB+SyzXhRcv884biAMPSDeUtcJupWlAQ3cORKI7TeU6FjObIpg+WwErgdDN24Ia0ypABUce4pUWTKQ7EMaf1C5Vji0WXnuauFLa/eZ9WLATlpfYftYhmHktVgJZf356412gHXFxl8a23KAEslhQOjDXb/UARZgN4d1QClYwKDAVzErLbIrjoDdVL6zDlBVy0nAUpJdLq2FGuVUAZ0zLQ1pvboqp8jiFJFcmp6c11w2lvluV+cQBLafF5yRCQMFNgllBoeKaabeZSSoOe9N67JWc5SmLSPbzdSzRSocgl1KYEMREF+6FTBpcTrcPi+YdTdh5FLlbHxL2hiDCtPbIakpA9EQ0vnlnGmtpW6Iz5XYwmE/PezXHjFIoINcFC4QcMGI41hXukIc+LVehkPOg40BPVVmjOpQLOuMY2UCONc0OdMcC6/nB59LjLvLyvmII8eD65Rpcj637IXA1XVR6TAMTXaT3x2GrvCW7ysLr+GjZO4qcTTKsrABLoP+Oq2V4yygANP2gROdYKbX57IrpVAFTqlOFDkBKYhmIMzgTFljq6N6ATTXjm/gdVUJtZQG0SgNGUEDGms+EzNgPDz8OcrzsWT2XPYosFnZiRzcdJT9Inenduj6UJn5iCSDWFWkC7leLXw2FlLzkHB1E5UchOZAM8XmPhUxyydygJn3GqCmUpUErKTTUmGP901lSTp7yrOr5vOdVDbrr7hegyDweopZqejKZMLyfCwNiu2DyFrifqPKVGxpUHGcIEkaWK0qXLdpJ56562Y88z4jjvDpI05C0V0ANlagglAkmcMwRLPZwdzcVqntouDNwpZNOPX4zTj7Ox/FgZe9DIM/+ANsed/7sP3KKzH/zj/DuNPBDztd9Ff7GF43wv6H9kAdPISd130PXzz+bPz9cefgxTd9FRc8dAc+ve88FIsmANCJhgoUVKAwG5Qo0tTUsWiN5riP9TBGFJkG9EVRQU0CAc+0zRQM+mtY2RcjbSa45FOfg3+9+K1/hH+74t144qc/gqN/cAPe/uu/heqIRRwx28AL3vQ63Hfu+fjOS1+Oyz73Uey49nt445NfguHMLMaDJoajnjTXpdOTJEaxtSi0bUBaIssmAlzBA0PG46Gofu6+8yE8+1v1YBMAjnrHO7D8mtcgbaUiKjLJRuIs8jhVSmFeBSgT03MsDmN09AQr3B+6QhIZSmij0UZnahpRbOzg86+v95N6xe3X4q93nIK1IMCWtI1T77oFXzz+bPzTKRfi5bd9G2fdfxdu3n4sLrvlWlx2i6GCfnL7CVBzm9FaP4iyLKxISnVYQTxrnAhSVuLkMwNSlhNQ2IfgBfeCOVMrVFUhKnoM/CMvyGVwQQU7U4Po6hzFFth6TtL7gjBEjMT6KIHcg1LKCjcV4nvQFzH3X1lb6IKLPB+I3QjDCHmWo9IZtLY9nrRTgwsCR6ki48c/U/h7nybnfAwriFNVUJWlg3nONCX0qbRb2qDNiRxEtlVIKFRCBiplVaDUhdgRgl5+xt7cW+TZGidQ4lOX2URXzp/AAGNG/Kywr2nKfdNfIUhowONYQCTA1F0GVmZf5tRmWXgmsfaVdpkS9FxvAnBb2y1+UZyisP5HFMUSRJAFQTBAKQXF+S/N7yit7oISr0diafqfllWBUBF4ZHBTSWP3UhgRrtTlcAEYgMyyEABrrh0gztdNJkPRVSD1tCxNM3o2S668ejxSRBlQiliWtG9wgRbHz78nBSW1zmwYDesnV7qU9jbmXrOayBmD1591/VIHWBxMAJKJCgKnxuan8c1mgjU2ZiEVOpd6DWYsGKgATqmOAQkdT6E22dQmJ1R7EbGfYiXKobxu3MxexHFsN7Cro3DIWWWzGYUU0df431YAgwgKAxw/Eqfjw7oYjgWfT0MDNmDwURw/hUvOKtWHiNBwc1M6FHAN9ZgxhFI1B8zvp5A22tYoWcTFo7G5zFEsY+7GPkBVMTNppKRdGjhEFDWlgNunU3A+6BhTfEBU9awkNe+BWUvXPBmyDngfACzakyOKXIDkp6p9xIaBs28IlApRSrrbH2MaAJdNo+F3hisQVNchYKbIleILlSYI4VT5KMvKjAbkedgcObFOqjnE0jSu1eaIUk+lZI6kjkG7hsOcM621Z6RC+T2BDCKcDDCYSXb0mEoyX0Hg5GcZVDFIJQWFF4EOX/3THY5W0Uub4DBNWzbAcnx/vy6y05kV+h8DT2Zh/L3r98vzM3AUkHDr2VCBWAPiK4BFUSKiLFk2lh4+Ck61z3d8/T3ii78w6HM2LRfApsqdyqkOmcmNhN/OII7iKlybNYdP69p4Q2u5R160zZT5Na0e4tph7K9FlwkwwFNemB5bbPCoUeGfTrkQv7LL9aJ6/7FnYdBftXMTSbau2ezazK1VigsjNJtddGY6OPqMo3HcBadi/+VXY67VwpEXXQR1+eXADTeg/ad/itZ//AdOuPJK3H70Ztx/630Y9NewbwL8j0f/GtaUwnjYw19uOx7/uPU4BEWJdn+EtJkiG08QhCG6xQRv/vw/4IbNx+BfTn8cXnbPjThn3734rbMuxbqukGeH0GpPIwoDSJ+dqhD6cndqFtH6Bk645SZcf9GTcP2rfgvP/qPfw6Zd9+K1v/FsAMB1F12GcnYGaSNBMT2De886B4/4wudwxhc+BwD49qMvQtbtIlIKqU7RbHYw6K8h1yXYkiAKIyRJU4R5DArekDU
```

Comment:
What's encoded here?

Reply:
This is just an output image from NPEX that's displayed inline in the Jupyter notebook. If we decide to go with converting all of the Jupiter notebooks to python scripts then this should dissapear